sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1979,6 +1979,14 @@ object SQLConf {
       .doc("When true, the ArrayExists will follow the three-valued boolean logic.")
       .booleanConf
       .createWithDefault(true)
+
+  val IGNORE_DATA_LOCALITY =
+    buildConf("spark.sql.ignore.datalocality")
+      .doc("If it is set to true, Spark won't fetch the block locations for each file on " +
```

Comment:
nit: I won't use abbreviation in the doc. `won't` -> `will not`

Reply:
Fixed.